### DISCUSSION TITLE
Ray + Laravel + Javascript setup

### DISCUSSION THREAD

**Discussion by jonneroelofs** [spatie/ray#318] [Q&A]
Can anyone show me the correct setup to get ray to work from javascript for Laravel 8 + Mix 6 ?
The documentation on ray and javascript links to another project and is very minimal. I was able to install js-ray and require it from app.js but the browser keeps throwing errors that ray is undefined.

Cheers,

Jonne

**Comment by patinthehat** [comment]
@jonneroelofs Take a look at the [node-ray](https://github.com/permafrost-dev/node-ray#using-with-laravel-mix) package - there's a section on using it with Laravel Mix in the readme.
